release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -1257,6 +1257,12 @@ default.
 
 The default Jenkins Node.js agent has been upgraded to Node.js version 12.
 
+[discrete]
+[id="ocp-4-6-audit-logs-not-gathered-by-default"]
+==== Audit logs not gathered by default for `oc adm must-gather` command
+
+The `oc adm must-gather` command no longer collects audit logs by default. You must include the `--audit-logs` parameter in the `oc` command to gather audit logs.
```

Comment:
You should use `oc adm must-gather -- /usr/bin/gather_audit_logs` command to gather audit logs.